frontend/src/app/choreography/choreography.component.ts

```diff
@@ -219,4 +219,8 @@ export class ChoreographyComponent implements OnInit {
   removePersonFromCarpet(index: number): void {
     this.clearItem(this.choreography.frames[this.activeFrame].subframes[this.activeSubframe].grid[index]);
   }
+
+  changeNoteVisibility(): void {
```

Comment:
**suggestion:** could be inlined into the template like `(click)="areNotesShown = !areNotesShown"`